Docker/ods-api-swaggerui/alpine/Dockerfile

```diff
@@ -33,10 +33,13 @@ RUN wget -q -O /app/SwaggerUI.zip https://pkgs.dev.azure.com/ed-fi-alliance/Ed-F
     rm -f /app/SwaggerUI.zip && \
     dos2unix /app/*.json && \
     dos2unix /app/*.sh && \
-    chmod 500 /app/run.sh && \
+    chmod 500 /app/run.sh -- ** && \
+    chmod 600 /app/appsettings.* && \
+    chmod 600 /app/web.config && \
     rm -f /app/*.exe && \
     apk del unzip dos2unix && \
-    chown -R edfi /app
+    chown -R edfi /app && \
+    chmod -R 700 /app/wwwroot
```

Comment:
I think this whole RUN statement can be 
```
    RUN umask 0077 && wget -q -O /app/SwaggerUI.zip https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_apis/packaging/feeds/EdFi/nuget/packages/EdFi.Suite3.Ods.SwaggerUI/versions/${SWAGGER_VERSION}/content && \
    unzip /app/SwaggerUI.zip -d /app && \
    rm -f /app/SwaggerUI.zip && \
    dos2unix /app/*.json && \
    dos2unix /app/*.sh && \
    chmod 500 /app/run.sh && \
    chmod 600 /app/appsettings.* && \
    rm -f /app/*.exe && \
    apk del unzip dos2unix && \
    chown -R edfi /app 
```

That way, we leave run.sh as read an execute, and everything else with default permissions with the 0077 umask applied, removing group and everyone permissions leaving only owner permissions